packages/vue-compat/__tests__/misc.spec.ts

```diff
@@ -275,3 +275,17 @@ test('ATTR_ENUMERATED_COERCION', () => {
     )('contenteditable', 'foo', 'true'),
   ).toHaveBeenWarned()
 })
+
+test('ATTR_ENUMERATED_COERCION: true', () => {
```

Comment:
The test name should be more specific. Something like:
```suggestion
test('ATTR_ENUMERATED_COERCION, coercing "false"', () => {
```